Automattic/wp-calypso
# Multiple domains: Select & show it in the side bar

Related to https://github.com/Automattic/dotcom-forge/issues/4033, https://github.com/Automattic/dotcom-forge/issues/4032

## Proposed Changes

On the domain selection screen of the hero flow https://wordpress.com/start/user we want to allow the user to select and purchase multiple domains. The selected domains should be listed in the right-hand column.

## TO DO
- [x] Rebuild & add styles `Your domains` sidebar: https://github.com/Automattic/wp-calypso/pull/82579
- [x] Add filter to show only on selected flows.

## Testing Instructions

> [!IMPORTANT]  
> By default, in the development environment, the flag `domains/add-multiple-domains-to-cart` is enabled. You can disable it by changing `/config/development.json` and re-run `yarn start`

* Go to `/start/domains` (onboarding flow)
* Check if you can select multiple domains
* Check if the sidebar shows the selected domains (still WIP)
* Check if you can delete the domains from the sidebar
* Check if `Your Domains` sidebar disappears when no domains are selected
* Check selecting one or more domains
* Continue to checkout (selecting a paid plan) and check if all domains shows.
* Check without the flag if it works as expected
* Test other flows with and without the flag, it should work as usual, without multiple domains enabled.
